test/torchaudio_unittest/transforms/autograd_test_impl.py

```diff
@@ -0,0 +1,45 @@
+from parameterized import parameterized
+from torch.autograd import gradcheck, gradgradcheck
+import torchaudio.transforms as T
+
+from torchaudio_unittest.common_utils import (
+    TestBaseMixin,
+    get_whitenoise,
+)
+
+
+class AutogradTestCase(TestBaseMixin):
+    def assert_grad(self, transform, *inputs, eps=1e-06, atol=1e-05, rtol=0.001):
+        transform = transform.to(self.device, self.dtype)
+
+        inputs_ = []
+        for i in inputs:
+            i.requires_grad = True
+            inputs_.append(i.to(dtype=self.dtype, device=self.device))
+        assert gradcheck(transform, inputs_, eps=eps, atol=atol, rtol=rtol)
```

Comment:
nit: use self.assertTrue()

Reply:
Is there a reason that `assertTrue` is preferred?
I always think that the message returned by `assertTrue`, `AssertionError: False is not true` non sense, so I always use `assert`.

Reply:
I was mostly suggesting to use the `self.` variant of the assert instead of the default one.
The  reason is that it makes the test framework aware of it in a better way than catching the error from the assert. And it allows it to give better messages when running the whole test suite.